app/assets/stylesheets/buttons.scss

```diff
@@ -53,6 +53,24 @@
     color:            $white      ;
     background-color: $royal-blue;
     box-shadow:       $btn-box-shadow;
+    &.active {
+      box-shadow: 4px 2px 2px $gray-100;
+    }
+    &.back, &.delete, &.ok, &.btn-wide, &.btn-no {
+      border-width: 0;
+    }
+    &.btn-wide {
+      width: 300px;
+    }
+    &.btn-no{
+      width: 196px;
+    }
+    &.back {
+      background-color: $gray-200!important;
```

Comment:
add space pls here (just code readability  improvement). Should look like "$gray-200 !important"